check.py

```diff
@@ -522,30 +523,32 @@ def run_unittest():
     raise Exception("unittest failed")
 
 
+TEST_SUITES = OrderedDict([
+  ('help', run_help_tests),
```

Comment:
If I'm reading this correctly, we'd say `check.py help` at the command line, and that would run the help tests. As opposed to `check.py --help` which would print usage information for check.py.

How about `'help-messages'` (or `'test-help-messages'`) to disambiguate?

Reply:
Done!